### DISCUSSION TITLE
Ambiguity in paper

### DISCUSSION THREAD

**Discussion by bruluk5w** [sebastian-hofstaetter/teaching#99] [Advanced IR: Exercise Q&A] [answered]
Dear Lecture Team,

Sorry if I have to ask this here, visible for everyone, since it concerns the solution for one of the exam questions. But since we are remote idk how to ask else to get a quick answer.
I think the paper for the exam contains a critical ambiguity:
Table 4 refers to the workflow BOW²>CLS as workflow 2, but figure 3 referes to CLS>BOW² as workflow 2. The text later sais that the paper will continue using workflow 2.
Now the question from the exam: Which one does it actually use?
My answer would be: I don't know, but unfortunately the multiple choice options do not include this answer.

Best,

LUkas

**Comment by sophiaalthammer** [comment, marked as answer]
Hi Lukas,

thanks for asking that and pointing it out for everyone! :) So workflow 2 is CLS>BOW^2, workflow 3 is BOW^2>CLS.

Best,
Sophia

**Reply by bruluk5w** [reply]
Thanks
